lib/pure/hashes.nim

```diff
@@ -123,6 +123,33 @@ proc hiXorLo(a, b: uint64): uint64 {.inline.} =
     else:
       result = hiXorLoFallback64(a, b)
 
+when defined(js):
+  import std/jsbigints
+  import std/private/jsutils
+
+  proc noBigInt(): bool =
```

Comment:
move to either `std/jsbigints` or `jsutils` and define it as: `proc hasJsBigInt*(): bool`